### ISSUE TITLE
[1.16.3] Server crash upon death with Totem of Undying in the inventory.

### ISSUE THREAD

**Issue by Nullmaruzero** [svenhjol/Charm#339]
I am getting a consistent, immediate server crash upon death with the Totem of Undying in the inventory. Either equipped in the hand or just in the backpack. We have tried different scenarios of death as well as places or gamerules.

Snippet the from crash-log:
```
Time: 10/22/20, 1:15 AM
Description: Exception in server tick loop

java.lang.NoSuchMethodError: 'int net.minecraft.class_1661.method_7395(net.minecraft.class_1799)'
	at svenhjol.charm.module.UseTotemFromInventory.tryFromInventory(UseTotemFromInventory.java:24)
	at net.minecraft.class_1309.redirect$zik000$hookTryUseTotem(class_1309.java:5327)
	[ ... ]
```
`keepInventory` is by default true, but we have also tried with false. The only other non-default gamerules I have is `mobGriefieng true` and `doInsomnia false`

Charm's config is default, haven't changed a thing there.

Mod version: `charm-fabric-1.16.3-2.0.3`
Java:
```
openjdk version "11.0.8" 2020-07-14
OpenJDK Runtime Environment (build 11.0.8+10-post-Ubuntu-0ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.8+10-post-Ubuntu-0ubuntu118.04.1, mixed mode, sharing)
```

[Crashlog — crash-2020-10-22_01.15.42-server.txt](https://github.com/svenhjol/Charm/files/5419239/crash-2020-10-22_01.15.42-server.txt)

**Comment by svenhjol** [comment]
Thanks for the report, I can repeat it locally and it will be fixed in 2.1.0 which is currently undergoing testing.

**Comment by svenhjol** [comment]
Here's a build of 2.0.3 with the fix applied. https://github.com/svenhjol/Charm/releases/download/2.0.3-fabric/charm-fabric-1.16.3-2.0.3-hotfix1.jar
